customisation/back-translation.md

```diff
@@ -1,6 +1,18 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Back-translation
-description:
+description: Back-translating or back-copying target language sentences to augment parallel data
 ---
+
+**Back-translation** is the process of translating the monolingual data in the target language into the source language and then back into the target language.
```

Comment:
I'd simplify "the monolingual data in the target language" to make it less wordy.

What do you think of something like this?

**Back-translation** is the process of using machine translation to translate again a machine translation output to generate synthetic data.

### Steps

1- First, the machine translation system translates a text from one language to another language.
2- Then, the system uses the machine translation output as input, and translates it back into the original language.
3- Finally, the system translates the resulting synthetic input text again into the output language.

Reply:
Do you think creating an image would make it easier to understand?

Reply:
I disagree with this description, because it makes it sound like then it is translated from source to target.  But the training data is only translated from target to source.

Translation from source to target only happens at inference time, but that's a given.